### PR TITLE
Looser contracts

### PR DESCRIPTION
**Pull request by agrapine**

### CODE REVIEW

File: lib/src/bloc/stated.dart [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import 'package:flutter/foundation.dart';
 import 'package:stated/src/core/core.dart';
 
-abstract class Stated<T> extends Observable implements ValueListenable<T> {
+///
```
Comment (pawelpetruch) [inline lib/src/bloc/stated.dart:4]: Some docu would be nice

**Comment by pawelpetruch** [comment]
With ListenableBuilder isn't Observer now redundant?

**Comment by pawelpetruch** [comment]
What is Serializer class for? 🤔

**Comment by agrapine** [comment]
> With ListenableBuilder isn't Observer now redundant?

Indeed. Dropped.

**Comment by agrapine** [comment]
> What is Serializer class for? thinking

WIP :D

**Pull request merged**